### DISCUSSION TITLE
Consistent face generation

### DISCUSSION THREAD

**Discussion by vansaj0701** [lllyasviel/Fooocus#3402] [Q&A]
I made a face using fooocus 1 month ago and that time it is giving same face model but now it is giving another model face. I use metadata from my original and set all setting according to it using Image Prompts metadata feature. It give same image or face when I generate but when I generate image using face swap or by changing some text prompt like clothes, it give me another model face. How can I fix it. I want to make AI instagram influencer like aitana lopez. OR can anyone suggest another free and open source tool for this. However fooocus is best. But I only have a problem with face consistency. I use default setting and use face swap for original image and pyracanny for target image.

**Comment by mashb1t** [comment]
> that time it is giving same face model but now it is giving another model face

> It give same image or face when I generate

Which of the two is true, does it work or not? Due to how Stable Diffusion works, the slightest changes to the prompt can have a high impact in the image generation (=> the face). If you want to generate the same face multiple times use the face you want to have as FaceSwap image prompt or, given you have 10+ images with this face, you may also train a LoRA for it.

**Reply by vansaj0701** [reply]
It is not working. With same FaceSwap image prompt, it us giving different face.
